e2e/process-services-cloud/edit-task-filters-component.e2e.ts

```diff
@@ -284,4 +290,20 @@ describe('Edit task filters cloud', () => {
         await expect(await tasksCloudDemoPage.editTaskFilterCloudComponent().editTaskFilterDialog().getFilterName()).toEqual('My Tasks');
         await tasksCloudDemoPage.editTaskFilterCloudComponent().editTaskFilterDialog().clickOnCancelButton();
     });
+
+    async function createNewCustomFilter(name: string) {
```

Comment:
can you add the return type Promise<void> ?

Reply:
return type added